prepare-videochat.sh

```diff
@@ -553,5 +567,10 @@ pactl set-default-source ${DEFAULT_SOURCE}
 pactl unload-module ${ECANCEL_ID}
 pactl unload-module ${SINK_ID}
 
+if [ $DISABLE_PROXY = 1 ]; then
+    # Renabling proxy on completion of the script
+    export http_proxy=$temp_http_proxy
+fi
+
```

Comment:
I don't think you need to restore the original value - the child process cannot change the environment of the parent process. We can omit this whole if..fi block.